.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: objective-c
+
+osx_image: xcode7.1
+
+env:
+global:
+- LC_CTYPE=en_US.UTF-8
+- LANG=en_US.UTF-8
+
+before_script:
+- gem install cocoapods --no-rdoc --no-ri --no-document --quiet
+- gem install xcpretty --no-rdoc --no-ri --no-document --quiet
+
+script:
+- xcodebuild clean build test -workspace AGSnapshotHelper.xcworkspace -scheme AGSnapshotHelper -sdk iphonesimulator -destination "platform=iOS Simulator,OS=9.1,name=iPhone 6s" | xcpretty -c
```

Comment:
I think this will be helpful, but should be added with code coverage feature (eg. with CodeCov integration).